src/extractFromJS.ts

```diff
@@ -0,0 +1,21 @@
+export function findTaggedTemplateLiteralsInJS(jsCode: string, tag: string): string[] {
+  // m for multiline
+  // g for matching multiple results
+  // capture the text inside the template literal with parentheses
+  const regex = new RegExp(tag + '\`([\\s\\S]+?)\`', 'mg');
```

Comment:
This regex wouldn't work if you put a space between the `gql` and the backticks within your source code which seems to be supported in Chrome and in [the spec](http://www.ecma-international.org/ecma-262/6.0/#sec-template-literals). I guess we can ignore this possibility?